kbase-extension/static/kbase/js/widgets/function_output/kbaseGenomeComparisonViewer.js

```diff
@@ -94,20 +94,20 @@ define (
 					"Genome","Legend"
 				];
 				for (var i in genomes) {
-					headings.push("G"+i);
+					headings.push("G"+(i+1));
 				}
 				tableGenomes.append('<tr><th><b>'+headings.join('</b></th><th><b>')+'</b></th></tr>');
 				for (var i in genomes) {
             		var genome = genomes[i];
             		var row = [
-            			"<b>G"+i+"</b>-"+genome.name,"# of families:<br># of functions:"
+            			"<b>G"+(i+1)+"</b>-"+genome.name,"# of families:<br># of functions:"
             		];
             		for (var j in genomes) {
             			var compgenome = genomes[j];
             			if (genome.genome_similarity[compgenome.genome_ref]) {
             				row.push(genome.genome_similarity[compgenome.genome_ref][0]+'<br>'+genome.genome_similarity[compgenome.genome_ref][1]);
             			} else if (j == i) {
-            				row.push(genome.families+'<br>'+genome.functions);
+            				row.push("<font color='#d269e'>" + genome.families+'<br>'+genome.functions + "</font>");
```

Comment:
this probably needs another character - should have 6 hex chars for a color

Reply:
Thanks Bill (@briehl). I was sure I copy and pasted that color from the other viewer. I guess not.